pandas/core/window/ewm.py

```diff
@@ -35,13 +35,14 @@ class EWM(_Rolling):
     ----------
     com : float, optional
         Specify decay in terms of center of mass,
-        :math:`\alpha = 1 / (1 + com),\text{ for } com \geq 0`.
+        :math:`\alpha = 1 / (1 + com)`, for :math:`com \geq 0`.
     span : float, optional
         Specify decay in terms of span,
-        :math:`\alpha = 2 / (span + 1),\text{ for } span \geq 1`.
+        :math:`\alpha = 2 / (span + 1)`, for :math:`span \geq 1`.
     halflife : float, optional
         Specify decay in terms of half-life,
-        :math:`\alpha = 1 - exp(log(0.5) / halflife),\text{for} halflife > 0`.
+        :math:`\alpha = 1 - \exp\left(-\ln(2) / halflife\right)`, for
```

Comment:
Is this change in the formula based on the code?

Reply:
The code uses `1 - np.exp(np.log(0.5) / halflife)`.
I just felt that `-ln(2)` is aesphetically more pleasing than `log(0.5)`.

Reply:
I don't have a preference, I was a bit confused about the change of the base of the logarithm. I think the original one can be misleading, I think it's possible to assume base 10.

In any case, your version looks good.